diff --git a/bootstrap/stage0/runtime/CMakeLists.txt b/bootstrap/stage0/runtime/CMakeLists.txt
index 7e1aa035..5e8ee7e5 100644
--- a/bootstrap/stage0/runtime/CMakeLists.txt
+++ b/bootstrap/stage0/runtime/CMakeLists.txt
@@ -25,7 +25,7 @@ if (NOT serenity_POPULATED)
 endif()
 
 file(REMOVE_RECURSE ${CMAKE_CURRENT_SOURCE_DIR}/AK)
-file(COPY ${serenity_SOURCE_DIR}/AK DESTINATION ${CMAKE_CURRENT_SOURCE_DIR})
+file(COPY ${serenity_SOURCE_DIR}/AK DESTINATION ${CMAKE_CURRENT_SOURCE_DIR} NO_SOURCE_PERMISSIONS)
 
 configure_file("AK/Debug.h.in" "${CMAKE_CURRENT_SOURCE_DIR}/AK/Debug.h")
 
diff --git a/runtime/CMakeLists.txt b/runtime/CMakeLists.txt
index 7e1aa035..5e8ee7e5 100644
--- a/runtime/CMakeLists.txt
+++ b/runtime/CMakeLists.txt
@@ -25,7 +25,7 @@ if (NOT serenity_POPULATED)
 endif()
 
 file(REMOVE_RECURSE ${CMAKE_CURRENT_SOURCE_DIR}/AK)
-file(COPY ${serenity_SOURCE_DIR}/AK DESTINATION ${CMAKE_CURRENT_SOURCE_DIR})
+file(COPY ${serenity_SOURCE_DIR}/AK DESTINATION ${CMAKE_CURRENT_SOURCE_DIR} NO_SOURCE_PERMISSIONS)
 
 configure_file("AK/Debug.h.in" "${CMAKE_CURRENT_SOURCE_DIR}/AK/Debug.h")
 
